src/form/Form.tsx

```diff
@@ -473,6 +506,12 @@ function Form({
     Object.assign(fieldValues, newFieldValues);
     // Always rerender from empty state for display purposes
     if (rerender || empty) setRender((render) => !render);
+
+    // Should the system validate on each field change?
```

Comment:
is this question addressed now that we're debouncing validations?

Reply:
This question is just my unorthodox way of commenting on the meaning of the if statement below it.  Since causing a concern/question by it presence, I will change it.